.yamllint

```diff
@@ -31,7 +33,7 @@ rules:
   line-length:
     # Increase from default of `80`
     # Based on https://github.com/PyCQA/flake8-bugbear#opinionated-warnings (`B950`)
-    max: 150
+    max: 88
```

Comment:
We've managed to keep this at `88` across all formulas.  We can always fall back to using a `yamllint` ignore for each line that really has to be over that number of characters.  Since we're ignoring the specific files anyway, this can be reset to the standard value.

Reply:
Thanks, that makes perfect sense.